modules/tensor_mechanics/src/kernels/StressDivergenceRSphericalTensors.C

```diff
@@ -27,19 +27,13 @@ StressDivergenceRSphericalTensors::StressDivergenceRSphericalTensors(const Input
 Real
 StressDivergenceRSphericalTensors::computeQpResidual()
 {
-  Real div = 0;
+  mooseAssert(_assembly.coordSystem() == Moose::COORD_RSPHERICAL,
```

Comment:
We can do it in intialSetup().  I was just avoiding changing too much code but this change is worth it. I'll update and repush...